res/css/views/settings/tabs/user/_LabsUserSettingsTab.scss

```diff
@@ -19,7 +19,8 @@ limitations under the License.
         margin-top: 32px;
 
         .mx_SettingsFlag {
-            margin-right: 0; // remove right margin to align with beta cards
+            margin-right: 0; // remove right margin to al
```

Comment:
Truncated comment?

Reply:
> 

Yeah, I didn't comment on it :sweat_smile:

Reply:
Could you bring back the rest of the comment?

Reply:
Sure :)